### PR TITLE
Feature/add azure

### PR DESCRIPTION
**Pull request by abielawa**
- Added Azure AKS cluster configuration in _envcommon/aks.hcl
- Added Azure for Kafka, Postgresql and Rabbitmq queues
- Added Azure for Github Actions Workflow
- Configured Service Principal auth and added secrets in Github for Azure login action

### CODE REVIEW

File: terraform/live/rabbitmq/az/prometheus/terragrunt.hcl [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    cluster_ca_certificate = "dGVzdA=="
```
Comment (Sztigar) [inline terraform/live/rabbitmq/az/prometheus/terragrunt.hcl:18]: I'm wondering about this mock, why is it different than the `client_certificate` and `client_key`? If it doesn't have to be real key I think we should make the other mock's shorter so it doesn't generate unnecessary clutter.

File: terraform/live/postgresql/az/prometheus/terragrunt.hcl [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    cluster_ca_certificate = "dGVzdA=="
```
Comment (Sztigar) [inline terraform/live/postgresql/az/prometheus/terragrunt.hcl:18]: Same as above.

File: terraform/live/rabbitmq/az/rabbitmq/terragrunt.hcl [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    cluster_ca_certificate = "dGVzdA=="
```
Comment (Sztigar) [inline terraform/live/rabbitmq/az/rabbitmq/terragrunt.hcl:17]: Same as above.

File: terraform/live/kafka/az/prometheus/terragrunt.hcl [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    cluster_ca_certificate = "dGVzdA=="
```
Comment (Sztigar) [inline terraform/live/kafka/az/prometheus/terragrunt.hcl:18]: Same as above.

File: terraform/live/kafka/az/kafka/terragrunt.hcl [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    cluster_ca_certificate = "dGVzdA=="
```
Comment (Sztigar) [inline terraform/live/kafka/az/kafka/terragrunt.hcl:17]: Same as above.

File: terraform/live/postgresql/az/postgresql/terragrunt.hcl [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    cluster_ca_certificate = "dGVzdA=="
```
Comment (Sztigar) [inline terraform/live/postgresql/az/postgresql/terragrunt.hcl:17]: Same as above.

File: README.md [hunk 7]
```diff
@@ -4,17 +4,22 @@
 ## Prerequisites
 1. Install [Terraform](https://www.terraform.io/) version 0.38.7 or newer.
 2. Install [Terragrunt](https://terragrunt.gruntwork.io/) version v1.2.6 or newer.
-3. Install [Python](https://www.python.org/) version v3.10 or newer.
-4. Install [jsonpath-ng](https://pypi.org/project/jsonpath-ng/) version v1.5.3 or newer.
+3. Install [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl).
```
Comment (Sztigar) [inline README.md]: Do we need kubectl installed locally? 🤔

File: terraform/live/rabbitmq/az/prometheus/terragrunt.hcl (hunk 1 above)
Comment (abielawa) [reply terraform/live/rabbitmq/az/prometheus/terragrunt.hcl:18]: The kubectl or helm provider checks whether the client_key variable has the value for PEM format private key. So even the mock value needs to be in PEM format. The same goes for client_certificate variable. Only the cluster_ca_certificate can be a short random encoded string.

File: terraform/live/postgresql/az/prometheus/terragrunt.hcl (hunk 2 above)
Comment (abielawa) [reply terraform/live/postgresql/az/prometheus/terragrunt.hcl:18]: Explained above.

File: terraform/live/rabbitmq/az/rabbitmq/terragrunt.hcl (hunk 3 above)
Comment (abielawa) [reply terraform/live/rabbitmq/az/rabbitmq/terragrunt.hcl:17]: Explained above.

File: terraform/live/kafka/az/prometheus/terragrunt.hcl (hunk 4 above)
Comment (abielawa) [reply terraform/live/kafka/az/prometheus/terragrunt.hcl:18]: Explained above.

File: terraform/live/kafka/az/kafka/terragrunt.hcl (hunk 5 above)
Comment (abielawa) [reply terraform/live/kafka/az/kafka/terragrunt.hcl:17]: Explained above.

File: terraform/live/postgresql/az/postgresql/terragrunt.hcl (hunk 6 above)
Comment (abielawa) [reply terraform/live/postgresql/az/postgresql/terragrunt.hcl:17]: Explained above.

File: README.md (hunk 7 above)
Comment (abielawa) [reply README.md]: Fair point, for the deployment with python script it's not necessary.

File: terraform/live/rabbitmq/az/prometheus/terragrunt.hcl (hunk 1 above)
Comment (Sztigar) [reply terraform/live/rabbitmq/az/prometheus/terragrunt.hcl:18]: It's really weird that it doesn't check format of `cluster_ca_certificate`.

**Review by Sztigar — APPROVED** [review]

File: terraform/live/rabbitmq/az/prometheus/terragrunt.hcl (hunk 1 above)
Comment (abielawa) [reply terraform/live/rabbitmq/az/prometheus/terragrunt.hcl:18]: In AWS or GCP we also didn't have to create a valid mock certificate for `cluster_ca_certificate` variable so I guess it's just how the kubectl/helm provider works.

File: .github/workflows/ci.yaml [hunk 8]
```diff
@@ -5,15 +5,20 @@ on:
     branches:
       - "v2"
 
-env: 
+env:
```
Comment (hubertklusek) [inline .github/workflows/ci.yaml]: ci.yaml has changed name to terraform.yaml on v2, shouldn't it be changed here as well?

**Pull request merged**